### ISSUE TITLE
Autumnal Cleanup 2015 Reporting

### ISSUE THREAD

**Issue by StreakyCobra** [syl20bnr/spacemacs#3549]
This issue is used to report contributors' work on the [**:fallen_leaf: Autumnal Cleanup 2015**](https://github.com/syl20bnr/spacemacs/wiki/Autumnal-Cleanup-2015).

If you want to take part of this collective effort, you can read a presentation [here](https://github.com/syl20bnr/spacemacs/wiki/Autumnal-Cleanup-2015) and discuss it on [Gitter](https://gitter.im/syl20bnr/spacemacs).

***

**Please do not ask or discuss anything here.**

***

This page is only used for reporting progress as comments in the following format:

```
- [ ] #<ISSUE_NUMBER> <FLAG>
```

As for instance:

```
- [ ] #1 Valid
- [ ] #2 Wontfix
- [ ] #3 NRA
- [ ] #4 Can't reproduce
```

*The checkboxes will be used by repository maintainers to mark their progression on dealing with issues, so please do not check any issue yourself.*

***

Contributions:
- d12frosted: 321 (55.44%)
- StreakyCobra: 200 (34.54%)
- dsdshcym: 20 (3.45%)
- cpaulik: 10 (1.73%)
- CestDiego: 10 (1.73%)
- ralesi: 10 (1.73%)
- sooheon: 5 (0.86%)
- robbyoconnor: 3 (0.52%)

Overall progress: 579/598 (96.82%)

**Comment by d12frosted** [comment]
- [ ] #1631 Fixed in Develop
- [ ] #3171 Unsure - actually I would close it, because there is #1048 for discussing `structured-haskell-mode` and Spacemacs
- [ ] #3443 Wontfix - should be reported upstream, it's not related to Spacemacs

**Comment by robbyoconnor** [comment]
:eyes:

**Comment by d12frosted** [comment]
- [ ] #2134 Valid
- [ ] #1704 Can’t Reproduce
- [ ] #3503 Valid - PR proposed
- [ ] #1048 Valid
- [ ] #2142 Fixed in Develop
- [ ] #1718 Fixed in Develop
- [ ] #1493 Fixed in Develop
- [ ] #3173 Fixed in Develop

**Comment by StreakyCobra** [comment]
- [ ] #1203 NRA - Concern an old version of spacemacs (v0.101.2)
- [ ] #3464 Valid
- [ ] #1193 Valid - "Beginner friendly" label should be added
- [ ] #48 No Response - Last update 2014, should probably be closed
- [ ] #2949 Valid

**Comment by StreakyCobra** [comment]
- [ ] #595 Fixed in develop
- [ ] #2086 Valid - "Beginner friendly" label should be added
- [ ] #3520 Valid
- [ ] #2317 Unsure - One-line PR to bind a new keybinding, but no response and no documentation
- [ ] #2087 Valid - or Wontfix ?

**Comment by StreakyCobra** [comment]
- [ ] #2669 Valid
- [ ] #2106 Wontfix
- [ ] #1153 No Response
- [ ] #1114 Unsure - Discussion over?
- [ ] #1559 NRA

**Comment by StreakyCobra** [comment]
- [ ] #1944 Fixed in develop
- [ ] #2294 Unsure - No response, but there seems to be an easy fix. I'll ping the thread.
- [ ] #2063 Valid
- [ ] #2626 No Response + Can’t Reproduce
- [ ] #2313 Unsure - One-line PR to bind a new keybinding, but no response and no documentation

**Comment by CestDiego** [comment]
I love this idea. :)

**Comment by StreakyCobra** [comment]
- [ ] #1583 NRA - Since 0.104, backspace delete one character and `C-h` goes one level up.
- [ ] #1768 Can’t Reproduce - First case is not reproducable, second case is not a bug (as explained)
- [ ] #2432 Wontfix - Should be reported upstream
- [ ] #2947 Valid - But reported upstream, can maybe be closed?
- [ ] #3277 Fixed in develop

**Comment by StreakyCobra** [comment]
- [ ] #1103 Valid
- [ ] #2311 Valid - Should be labelled with: "Enhancement" and "Beginner friendly"
- [ ] #2600 Valid
- [ ] #2868 Wontfix - Reported upstream, should be closed.
- [ ] #3109 Valid

**Comment by CestDiego** [comment]
- [ ] #1357 Not verified
- [ ] #1831 Not verified
- [ ] #1973 Not verified
- [ ] #2198 Not verified
- [ ] #2459 Not verified
- [ ] #2562 Not verified
- [ ] #2575 Not verified
- [ ] #2708 Not verified
- [ ] #2841 Not verified
- [ ] #3254 Not verified

**Comment by StreakyCobra** [comment]
- [ ] #1818 Unsure - Can't reproduce, but I'm not used to C/C++ and I'm not sure to understand the bug
- [ ] #2089 Wontfix - Emacs default behaviour, and `vim-empty-lines` does it.
- [ ] #2360 Can't reproduce - Probably an installation specific bug, will ping the thread.
- [ ] #2648 Valid - But hanging thread, will ping to reactivate it.
- [ ] #2865 Valid - Should be labelled `OS X` and `Beginner Friendly`.

**Comment by d12frosted** [comment]
- [ ] #1853 Can't reproduce - but waiting for more input. Related issue was already closed.

**Comment by d12frosted** [comment]
- [ ] #2049 Can't reproduce – but the user that opened this issue found solution for this issue - using another package
- [ ] #2110 Can't reproduce - old issue
- [ ] #2315 Can't reproduce - but this doesn't look like spacemacs problem
- [ ] #2632 Valid - but can't reproduce it (also on El Capitan)
- [ ] #3268 NRA - my issue is not relevant!

**Comment by robbyoconnor** [comment]
We now have three sr-speedbar PRs. 
- [ ] #2002 
- [ ] #3224
- [ ] #3559

**Comment by StreakyCobra** [comment]
- [ ] #1118 Valid
- [ ] #2171 Fixed in develop
- [ ] #2930 Valid - Candidate to be merged!
- [ ] #3165 Wontfix - It's a more general problem addressed in #3472
- [ ] #3170 Valid

**Comment by StreakyCobra** [comment]
- [ ] #1304 Valid - And reproducable
- [ ] #2335 Valid - But #2342 will close it, once accepted
- [ ] #3128 Discussion over - Next step is an eventual PR from original author
- [ ] #3369 Valid
- [ ] #3549 :joy: I think this one is OK for now

**Comment by StreakyCobra** [comment]
- [ ] #751 Wontfix - `evil-shift-round`-related bug.
- [ ] #1521 Discussion over
- [ ] #2783 Fixed - Solution proposed
- [ ] #3393 Valid - Should probably be added to 0.105 milestone
- [ ] #3456 Valid

**Comment by StreakyCobra** [comment]
- [ ] #1206 Valid - Could also be labelled `Beginner friendly`
- [ ] #2057 Valid
- [ ] ~~#2501~~ Already Closed
- [ ] #3428 Unsure - Could maybe be closed for the more general #2113 issue?
- [ ] #3507 Valid

**Comment by d12frosted** [comment]
- [ ] #2338 Wontfix

**Comment by d12frosted** [comment]
- [ ] #3280 Valid - I will probably try this suggestion
- [ ] #3302 Valid
- [ ] #3370 Valid

**Comment by robbyoconnor** [comment]
I went ahead and closed all my WIP PRs -- I got a bit over-eager and then...life.

**Comment by d12frosted** [comment]
- [ ] #2476 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #2750 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #2826 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #2851 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3181 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3218 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3222 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3467 Fixed in develop - it's properly marked and will be closed automatically once commit is in master

**Comment by StreakyCobra** [comment]
- [ ] #559 Valid - Should be labelled as `Enhancement` (otherwise discussion over...?)
- [ ] #820 Valid
- [ ] #1204 Valid
- [ ] #1915 Valid - it's an UBO, so can't reproduce, though
- [ ] #2180 Valid
- [ ] #2351 NRA - Can't reproduce the bug on master, nor on develop
- [ ] #2732 Unsure - Valid or Discussion over?
- [ ] ~~#3206~~ Already closed

**Comment by d12frosted** [comment]
- [ ] #1133 Fixed in develop - I would believe TheBB, but after release needs to be manually closed
- [ ] #1296 Fixed in develop - needs to be manually closed
- [ ] #2161 Fixed in develop - needs to be manually closed
- [ ] #2371 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #2659 Fixed in develop - needs to be manually closed
- [ ] #3022 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3111 Fixed in develop - needs to be manually closed
- [ ] #3208 Fixed in develop - needs to be manually closed
- [ ] #3214 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3278 Fixed in develop - needs to be manually closed
- [ ] #3327 Unsure - Marked as Fixed in develop and while helm-flx is really in develop, it has some open issues, so not sure
- [ ] #3433 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3457 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3481 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3534 Fixed in develop - my testing shows that it's really fixed there (had this issue on Ubuntu) - needs to be manually closed
- [ ] #3459 Valid - should fix #3458
- [ ] #2211 Invalid / Should be closed - See my comment there.
- [ ] #2924 Valid - waiting for more input

**Comment by StreakyCobra** [comment]
- [ ] #1647 Valid
- [ ] #1958 Valid
- [ ] #1959 Unsure - Either Valid or Wontfix
- [ ] #2096 Valid
- [ ] #2332 Valid - Simple PR that could probably be merged once rebased
- [ ] #2435 Valid - Simple PR that could probably be merged once rebased
- [ ] #2560 Valid
- [ ] #2625 Can't reproduce - I asked the original poster if it's ok on his side
- [ ] #2845 Valid - The most discussed PR :tada:
- [ ] #2974 Valid - Bug reproduced

**Comment by d12frosted** [comment]
- [ ] #872  NRA: already fixed
- [ ] #1374 Unsure: about conventions
- [ ] #2090 Can't reproduce
- [ ] #2514 Valid
- [ ] #2828 Unsure: looks valid though
- [ ] #3012 Unsure: initial problem was solved and discussion went down to other issue (that is not really related to Spacemacs itself)
- [ ] #3018 Valid
- [ ] #3381 NRA: looks like it's already [merged](https://github.com/syl20bnr/spacemacs/commits/develop/layers/geolocation/README.org)
- [ ] #3065 Fixed in develop - it's properly marked and will be closed automatically once commit is in master
- [ ] #3290 Fixed in develop - needs to be manually closed

**Comment by StreakyCobra** [comment]
- [ ] #773 Unsure - VIMUSERS.org documents this, so can probably be closed.
- [ ] #1042 Unsure - I don't sufficiently understand the problem to reproduce it…
- [ ] #1331 Valid
- [ ] ~~#2234~~ Already Closed
- [ ] #2238 NRA - Environment specific bug
- [ ] #2651 Wontfix - Old bug, not reproduced (concerning last comment: `SPC f e h`open buffer in read-only, it's why there is no evil bindings for delete line)
- [ ] #2787 NRA - Old discussion, and same discussion as #2525 which is closed
- [ ] #3202 Fixed - And will be closed by #3557 if merged
- [ ] #3448 Wontfix - Except if you want to revert `C-h` in helm

**Comment by d12frosted** [comment]
- [ ] #341  Can't reproduce: seems working to me
- [ ] #2065 Valid
- [ ] #2097 Valid
- [ ] #2307 Valid: lacks tagging
- [ ] #2321 Valid: needs some attention from ruby-guys
- [ ] #2856 Valid: UFO!
- [ ] #2909 NRA: Looks like the problem wasn't a problem and issue opener solved his problems
- [ ] #3020 Unsure
- [ ] #3547 Valid

**Comment by StreakyCobra** [comment]
- [ ] #1977 Valid - If there is a quelpa label, it may be added to find back those layers when it's ready
- [ ] #2026 Fixed - (as said by the original poster)
- [ ] #2029 Valid
- [ ] #2170 Unsure - Valid or Wontfix or NRA?
- [ ] #2784 Valid
- [ ] #2791 No Response - 2 month old and nobody is looking at it. Maybe ping bogdanteleaga ?
- [ ] #2979 Valid
- [ ] #3027 Unsure - I don't understand if the OP is fine with syl20bnr answer?
- [ ] #3418 Valid

**Comment by StreakyCobra** [comment]
- [ ] #1391 Wontfix - I oriented him to `bepo` as an example.
- [ ] #1599 NRA - old version, nobody else seems to complain
- [ ] #1732 Fixed - if he is speaking about `/` with `n` and `N`
- [ ] #1797 Discussion over?
- [ ] #1835 Fixed upstream
- [ ] #1874 Valid - Bug reproduced
- [ ] #2888 Wontfix - Can't reproduce (but not much details in there...)
- [ ] #3236 Fixed
- [ ] ~~#3500~~ Already Closed
- [ ] #3532 Valid

**Comment by d12frosted** [comment]
- [ ] #561  Valid
- [ ] #776  Valid
- [ ] #848  Valid
- [ ] #1313 Unsure: eyebrowse is added, perspective is in WIP state, but discussion doesn't move
- [ ] #1516 Valid
- [ ] #2284 Valid: actually I encounter the same problem
- [ ] #2950 Valid: should be merged
- [ ] #3019 Valid: should be labeled
- [ ] #3152 Valid: requires some feedback I think
- [ ] #3203 Valid
- [ ] #3260 Already closed
- [ ] #3415 Valid: requires some feedback
- [ ] #3421 Valid
- [ ] #3493 Valid
- [ ] #3546 Valid: requires some feedback
- [ ] #2864 Unsure: I think it should be closed since it's expected, but probably we would like somehow remove that warning, cause it's not the first and not the second time when I tell people that it's ok to have that warning in terminal
- [ ] #780  Unsure: looks like valid, but I don't really understand the problem; also it has [Haskell] in title, but not related to it I think

**Comment by d12frosted** [comment]
- [ ] #1497 Valid
- [ ] #690 Valid
- [ ] #1182 Unsure - discussion is stuck
- [ ] #1819 Valid
- [ ] #1982 NRA
- [ ] #2172 Valid 
- [ ] #2277 Fixed in develop - and properly marked
- [ ] #2686 Unsure - issue needs some attention - at least label it
- [ ] #2850 Valid
- [ ] #3103 Valid
- [ ] #3529 Valid - needs some attention

**Comment by d12frosted** [comment]
- [ ] #884 Valid - but I am not sure if it's relevant
- [ ] #1059 Valid - for Haskell it's important
- [ ] #1425 Unsure - looks like irrelevant
- [ ] #1726 Valid - but it's an old thing and no one looked into it
- [ ] #2160 Valid - needs more attention
- [ ] #2512 NRA
- [ ] #2957 NRA - it has nothing to do with Emacs as issue starter says
- [ ] #3429 Valid - @StreakyCobra, open PR

**Comment by StreakyCobra** [comment]
- [ ] #329 NRA - Perspective layer is doing it.
- [ ] #1443 Unsure - `S-BACKSPACE` is `C-H` inside terminal. Related to #3472, discussion can be kept there maybe?
- [ ] #1455 Not verified [@StreakyCobra is waiting on emacs to compile to try reproducing it :zzz:)
- [ ] #1803 Valid - I reproduced it with success
- [ ] #2224 Fixed in develop
- [ ] #2334 Can't reproduce - Each time the focus is lost from mini-buffer, it exits.
- [ ] #2392 Valid - Candidate for merge
- [ ] #2929 Valid - Candidate for merge (:eye: for conflict with #3339)
- [ ] #3145 Can't reproduce
- [ ] #3330 Can't reproduce - See my comment in it.

**Comment by cpaulik** [comment]
- [ ] #517 Can’t Reproduce - works for me
- [ ] #2042 Unsure but leaning towards NRA
- [ ] #2299 Valid but outdated
- [ ] #2590 Unsure leaning towards Fixed
- [ ] #2943 Valid
- [ ] #3116 Discussion over - continued in https://github.com/syl20bnr/spacemacs/issues/3198
- [ ] #3158 Already closed
- [ ] #3420 Valid
- [ ] #3495 Valid
- [ ] #3543 Already closed

**Comment by StreakyCobra** [comment]
- [ ] #1097 Valid - Reproduced with success on my side
- [ ] #1598 Valid
- [x] ~~#1675~~ Already closed
- [ ] #2696 Valid - PR needing core reviewer with expertise
- [ ] #2823 Valid
- [ ] #2958 Valid - Candidate for merge?
- [x] ~~#3258~~ Already closed
- [ ] #3273 Valid - Could possibly be tagged as `Enhancement`
- [ ] #3321 Valid - (There are crazy people on this planet :joy:)
- [ ] #3471 Valid - To be sure, [this](https://github.com/syl20bnr/spacemacs/blob/develop/layers/+source-control/git/packages.el#L15) is just for packages, not layers, right?

**Comment by dsdshcym** [comment]
- [ ] #481  Not verified
- [ ] #1488 Not verified
- [X] #1591 No Response
- [X] #1989 Can't Reproduce - Wait for @lululau to confirm
- [X] #2145 Wontfix - `SPC f e h` has been used for a pretty long time. And @syl20bnr said he would keep `SPC f e h` in this thread
- [ ] #2274 Not verified
- [ ] #2324 Not verified
- [ ] #2872 Not verified
- [ ] #2990 Not verified
- [ ] #3548 Not verified

**Comment by dsdshcym** [comment]
- [ ] #1418 Valid
- [ ] #1533 Not Verified
- [ ] #1921 Not Verified
- [ ] #1922 Not Verified
- [ ] #1949 Not Verified
- [ ] #2033 Not Verified
- [ ] #2108 Not Verified
- [ ] #2832 Not Verified
- [ ] #3098 Not Verified
- [ ] #3259 Not Verified

**Comment by d12frosted** [comment]
- [ ] #2103 Unsure
- [ ] #2207 NRA
- [ ] #2316 Unsure: not labeled, solutions provided, probably needs more attention
- [X] #2968 Already closed
- [ ] #3136 Valid
- [X] #3408 Already closed
- [ ] #3476 Valid: needs more attention
- [ ] #3491 Valid: PR is proposed #3609
- [ ] #3502 Valid
- [ ] #3539 Fixed in develop: marked properly, but needs to be manually closed
- [ ] #2012 Unsure: don't quite understand the problem - when I run `debugger-mode` key bindings works fine for me

**Comment by TheBB** [comment]
I know I'm not supposed to comment here, but I just want to say thanks for the work you guys do here. I may be slow in going through this thread but I haven't forgotten it. :-)

**Comment by d12frosted** [comment]
- [ ] #1423 Valid
- [ ] #1745 Valid
- [ ] #1781 Valid
- [ ] #1965 No response - old topic
- [ ] #1992 Valid
- [ ] #2128 Valid - needs some attention from ruby guys
- [ ] #2652 Valid - needs to be merged
- [ ] #2619 Unsure
- [ ] #2722 Unsure - like previous
- [ ] #2767 Valid - and needs attention
- [ ] #2772 Valid
- [ ] #3110 Unsure - looks like it's fixed, but no response from OP
- [ ] #3282 Valid - add documentation label
- [ ] #3379 Valid
- [ ] #3484 Valid
- [ ] #3524 Valid - IMHO should be merged

**Comment by StreakyCobra** [comment]
- [ ] #1404 Unsure - Frames are already nice now. Should be labelled as `Enhancement` otherwise.
- [ ] #1417 NRA - See last comment from CestDiego
- [ ] #1474 Valid
- [ ] #1481 Valid - Should be labelled as `Beginner friendly`
- [ ] #1584 NRA - Seems an upstream bug. Should be labelled as `Microsoft Windows`
- [ ] #1978 Valid - Reproduced, should be reported upstream also
- [ ] #2020 Valid
- [ ] #2189 Fixed
- [ ] #2352 NRA - Installation specific bug, old.
- [ ] #2602 Fixed - `spaceline` replaced `powerline` 
- [ ] #2677 Valid - Candidate for merge
- [ ] #2725 Unsure - Valid and should be labelled `24.3` and `Org`, or NRA
- [ ] #2899 Unsure - Don't understand what the bug is. Maybe just an enhancement?
- [ ] #3090 Valid - Should be labelled as `Bug` and `OS X`. Waiting for answer from OP
- [ ] #3391 Valid - Until #3390 is solved
- [ ] #3438 Valid - Should be labelled as `Beginner friendly`
- [x] ~~#3486~~ Already closed
- [ ] #3512 Valid
- [ ] #3545 Valid - Candidate for merge

**Comment by sooheon** [comment]
- [ ] #2621 Can't reproduce (repro steps unclear)
- [ ] #2691 Can't reproduce (no steps given)
- [ ] #3001 Won't fix (mix of resolved, outside spacemacs scope. Should be closed)
- [ ] #3388 Unsure (solution is found, but how to incorporate into spacemacs? Should it be documented and left to user config?)
- [ ] #3424 Valid

**Comment by d12frosted** [comment]
- [ ] #255 Valid
- [ ] #605 Valid
- [ ] #702 Valid
- [ ] #1211 NRA: since which-key is used now
- [ ] #2291 Can't reproduce
- [ ] #2331 NRA: exec-path-from-shell is already used
- [ ] #2395 Can't reproduce
- [ ] #2702 Wontfix
- [ ] #2706 Valid: needs some attention
- [ ] #2837 Valid: needs some attention
- [ ] #335 Valid

**Comment by d12frosted** [comment]
- [ ] #989 NRA
- [ ] #1222 Valid: not finished
- [ ] #1955 Valid
- [ ] #2114 Already closed
- [ ] #2135 Valid
- [ ] #2214 Valid
- [ ] #2545 Valid
- [ ] #3450 Valid
- [ ] #3475 Valid
- [ ] #3477 Valid

**Comment by d12frosted** [comment]
- [ ] #1258 Valid
- [ ] #1362 Unsure - too broad and old
- [ ] #2157 Valid
- [ ] #2781 Unsure: looks like there is no interest in this PR
- [ ] #3237 Can't reproduce
- [ ] #3362 Fixed in develop: properly labeled, but needs to be closed manually
- [ ] #3542 Valid
- [ ] #1846 Valid: though some of them are working for me
- [ ] #2456 Valid: should be labeled
- [ ] #2494 Valid

**Comment by d12frosted** [comment]
- [ ] #774 Valid
- [ ] #1088 NRA: it was fixed
- [ ] #1177 Valid
- [ ] #1359 NRA: we already have it in changelog, right?
- [ ] #1634 NRA: doesn't sound like an issue and also c++ layer has notes about semantics layer
- [ ] #1823 NRA
- [ ] #2163 Valid
- [ ] #2366 NRA: looks like it's fixed in `evil-escape`
- [ ] #2369 Fixed in develop
- [ ] #2698 Valid: needs some attention
- [ ] #2713 Fixed in develop
- [ ] #2785 Valid: needs some attention
- [ ] #2813 NRA: fixed upstream
- [ ] #3070 Valid
- [ ] #3087 Valid
- [ ] #3178 Valid: kind of UFO issue
- [ ] #3233 Valid
- [ ] #3402 Valid
- [ ] #3525 Valid
- [ ] #3528 Valid

**Comment by d12frosted** [comment]
- [ ] #910 Unsure
- [ ] #1504 NRA: question is answered
- [ ] #1588 Valid
- [ ] #1759 Unsure: a little bit abandoned
- [ ] #1927 Unsure: old question and looks like a wont fix
- [ ] #2064 Valid
- [ ] #2162 Valid
- [ ] #2199 Valid
- [ ] #2342 Valid: candidate for merge
- [ ] #2495 Valid
- [ ] #2517 Unsure: looks like a wont fix
- [ ] #2635 Valid: needs some attention
- [ ] #2715 Unsure: looks like it's already fixed
- [ ] #2747 Valid: needs more real attention
- [ ] #2840 Wontfix and no answer from OP
- [ ] #3067 Valid: but lacks activity from OP
- [ ] #3226 Valid
- [X] #3490 Already closed
- [ ] #3531 Valid
- [X] #3544 Already closed

**Comment by StreakyCobra** [comment]
- [ ] #512 Unsure - Valid or Wontfix
- [ ] #1281 Unsure - perspective potentially replaces the need of having purposes?
- [ ] #1330 NRA - orphan deletion is documented, as same as additional packages and layers
- [ ] #1758 Valid
- [ ] #2006 Valid - Candidate to merge if of interest, otherwise can be closed.
- [ ] #2043 Fixed upstream - Can be closed: `syl20bnr: The PR has been merged upstream.`
- [ ] #2104 No response | Discussion over
- [x] ~~#2196~~ Already closed
- [ ] #2292 No response
- [ ] #2349 Valid - OP pinged, waiting for response
- [ ] #2736 Valid - Reproduced, can be labelled `Beginner's friendly`
- [ ] #2834 Valid
- [ ] #3125 Valid - But probably an upstream bug
- [ ] #3198 Valid
- [ ] #3232 NRA - Can't reproduce on `0.104.3`, pinged, waiting for response.
- [ ] #3422 Valid
- [ ] #3472 Valid
- [ ] #3478 Wontfix - Change already included in – discussed  and more general – #3384
- [ ] #3506 Valid - Maybe `Beginner's friendly` as it seems just to add a package to chinese layer.
- [ ] #3535 Wontfix - `evil-escape` can be blacklisted, and justbur may integrate it in the future.

**Comment by d12frosted** [comment]
- [ ] #483 Valid
- [ ] #586 Valid
- [ ] #896 Valid
- [ ] #1165 Valid
- [ ] #1217 Fixed in develop: lacks appropriate tag, but will be closed automatically once 8379c2f is merged
- [ ] #1303 Valid
- [ ] #1333 Valid
- [ ] #1394 NRA: already fixed
- [ ] #1574 Valid
- [ ] #1748 Fixed upstream
- [ ] #1918 Unsure: looks like it's still valid, but I am not sure
- [ ] #2113 Valid
- [ ] #2222 Valid
- [ ] #2533 Fixed in develop: properly labeled, but must be closed manually after major release
- [ ] #2714 Valid
- [ ] #2869 Valid
- [ ] #2887 Valid: needs some attention from elisp users
- [ ] #2988 Unsure: looks like Wontfix - author of package said that this behavior is intended
- [ ] #3112 Valid: requires some investigation
- [ ] #3441 Valid

**Comment by d12frosted** [comment]
- [ ] #247 Valid
- [ ] #299 Valid: work in progress in #2652
- [ ] #891 Valid
- [ ] #1008 NRA: looks like it's fixed by #1009
- [ ] #1167 Valid
- [ ] #1265 Unsure: can't reproduce, no response for a long time
- [ ] #2075 NRA: should be closed
- [ ] #2105 Valid
- [X] #2174 Already closed
- [ ] #2250 Valid
- [ ] #2280 Unsure: issue is old, no response
- [ ] #2337 Fixed in develop: marked properly, but needs to be closed manually
- [ ] #2355 Valid
- [ ] #2633 Valid
- [ ] #2673 Unsure: sounds like wont fix
- [ ] #2699 Valid: needs some attention
- [ ] #2704 Valid: reported upstream, but still not fixed - looks like there is no interest in fixing it
- [ ] #3014 Unsure: @TheBB, so what have you decided?
- [ ] #3284 Valid
- [X] #3431 Already closed

**Comment by d12frosted** [comment]
- [ ] #480 Valid: easy to fix
- [ ] #1067 NRA: we already have change log in spacemacs home buffer
- [ ] #1245 NRA: already documented on master (in README)
- [ ] #1326 NRA: looks like it's fixed upstream
- [ ] #1837 Valid: easy to fix
- [ ] #2136 Unsure: looks like we can't help with it and upstream is not interested in fast solving this problem
- [X] #2304 Already closed
- [ ] #2448 Valid
- [ ] #2593 Unsure: is it even relevant?
- [ ] #2615 Can't reproduce
- [ ] #2753 Valid
- [ ] #2835 Valid
- [ ] #2880 Valid
- [ ] #3120 Valid
- [ ] #3242 Valid: PR is proposed (#3248)
- [ ] #3248 Valid: no activity
- [ ] #3338 Valid
- [ ] #3366 Unsure
- [X] #3454 Already closed
- [ ] #3501 Fixed in develop: marked properly, but needs to be closed manually

**Comment by d12frosted** [comment]
- [ ] #801 Unsure: no activity
- [ ] #823 Valid
- [ ] #1269 Valid: but it's irrelevant to Spacemacs
- [ ] #1445 NRA
- [X] #1469 Already closed
- [ ] #1522 Valid
- [ ] #1544 Valid
- [ ] #1742 Valid: beginner friendly
- [ ] #1816 Unsure
- [ ] #1856 Unsure: looks like it's NRA
- [ ] #1863 Wontfix: TheBB proposed possible solution for this, but looks like we don't want it in Spacemacs
- [ ] #1869 Unsure: waiting for response
- [ ] #1928 Fixed in develop: marked properly, but needs to be closed manually
- [ ] #2088 Fixed in develop: must be labeled accordingly, will be closed automatically after release
- [ ] #2118 Unsure: waiting for response
- [ ] #2144 Valid: needs someone to implement it
- [ ] #2147 Unsure: quite old and don't think that someone will reproduce it now

**Comment by StreakyCobra** [comment]
- [ ] #1298 Not verified
- [ ] #1730 Not verified
- [ ] #2278 Not verified
- [ ] #2329 Not verified
- [ ] #2452 Not verified
- [ ] #2671 Not verified
- [ ] #2843 Not verified
- [ ] #3378 Not verified
- [ ] #3462 Not verified
- [ ] #3538 Not verified

**Comment by StreakyCobra** [comment]
- [ ] #860 Not verified
- [ ] #903 Not verified
- [ ] #1867 Not verified
- [ ] #1950 Not verified
- [ ] #2231 Not verified
- [ ] #2254 Not verified
- [ ] #2522 Not verified
- [ ] #2710 Not verified
- [ ] #3407 Not verified
- [ ] #3509 Not verified

**Comment by StreakyCobra** [comment]
- [ ] #1181 Taken care directly
- [ ] #1255 Taken care directly
- [ ] #1520 Taken care directly
- [ ] #1687 Taken care directly
- [ ] #1855 Taken care directly
- [ ] #2272 Taken care directly
- [ ] #2775 Taken care directly
- [ ] #3079 Taken care directly
- [ ] #3108 Taken care directly
- [ ] #3252 Taken care directly

**Comment by d12frosted** [comment]
- [ ] #2236 Not sure: looks like it's NRA though
- [ ] #2264 NRA
- [ ] #2273 NRA: proposed solution is fine, nothing else to do
- [ ] #2323 Valid
- [X] #2361 Already closed
- [X] #2376 Already closed
- [ ] #2571 Reported upstream
- [X] #2623 Already closed
- [ ] #2946 Valid ?
- [X] #2966 Already closed
- [ ] #3039 Invalid: proposed solution, no response - should be closed
- [ ] #3137 Can't reproduce
- [ ] #3244 Fixed in develop by #3257
- [ ] #3307 Fixed in develop: properly marked but need to be closed manually
- [X] #3319 Already closed
- [ ] #3354 Valid: PR proposed, but is not finished
- [ ] #3384 Unsure: discussion is run out
- [ ] #3390 Fixed in develop: properly marked but need to be closed manually
- [ ] #3395 Valid
- [ ] #3430 Valid
- [ ] #3480 Valid

**Comment by StreakyCobra** [comment]
- [ ] #1778 Taken care directly
- [ ] #1976 Taken care directly
- [ ] #2151 Taken care directly
- [ ] #2327 Taken care directly
- [ ] #2654 Taken care directly
- [ ] #2749 Taken care directly
- [ ] #2937 Taken care directly
- [ ] #3212 Taken care directly
- [ ] #3251 Taken care directly
- [ ] #3440 Taken care directly

**Comment by d12frosted** [comment]
- [ ] #621 PR proposed, but probably wont fix
- [ ] #1731 Valid
- [ ] #1779 Unsure: since there is active work on new site this might be wont fix :-P
- [ ] #1894 Valid: WIP
- [ ] #1986 Already fixed: duplicate of #1314
- [ ] #2210 Valid: probably beginner friendly
- [ ] #2563 Valid: but it's not on our side
- [ ] #2720 Unsure: asked about the status of issue
- [ ] #2776 Valid: needs attention
- [ ] #2838 Unsure: looks like we don't use ProjectTalk
- [ ] #2874 Valid: needs attention
- [ ] #2886 Valid
- [X] #3043 Already closed
- [ ] #3064 Valid: but it lacks attention
- [ ] #3114 Valid
- [ ] #3263 Valid
- [ ] #3426 Valid: needs some attention
- [ ] #3439 Unsure: there is a solution for this problem, but should it be solved somewhere (spacemacs, upstream)?
- [ ] #3514 Valid
- [X] #3533 Already closed (totally VALID :-P )

**Comment by d12frosted** [comment]
- [ ] #3066 Valid
- [ ] #2520 Valid

**Comment by ralesi** [comment]
- [ ] #1253 Discussion Over - No response
- [ ] #1410 Valid, pr suggestions for bindings
- [ ] #1666 3rd party fix suggested in Issue
- [ ] #1850 Discussion open.  Long term planning
- [ ] #2140 Wontfix, standard emacs binding.
- [ ] #2350 Already Closed
- [ ] #2663 NRA with switch to spaceline
- [ ] #3041 Valid, can create PR for toggle.
- [ ] #3250 Already Closed
- [ ] #3436 Unsure, don't have ruby dev environment to test out.

**Comment by d12frosted** [comment]
- [ ] #232 Valid
- [ ] #1830 Valid
- [ ] #1834 NRA: already fixed by spaceline
- [ ] #1998 Invalid
- [ ] #2271 Valid
- [ ] #2554 Valid: needs some attention!
- [ ] #2667 Unsure: solution proposed, but should it be added to spacemacs?
- [ ] #2723 Valid
- [ ] #3204 Fixed in develop: marked properly, but needs to be closed manually
- [X] #3377 Already closed
- [ ] #3458 PR proposed: #3459
- [ ] #3497 Valid

**Comment by StreakyCobra** [comment]
- [ ] #3526 Taken care directly
- [ ] #3465 Taken care directly
- [ ] #3355 Taken care directly
- [ ] #3460 Taken care directly
- [ ] #3193 Taken care directly
- [ ] #2209 Taken care directly
- [ ] #3405 Taken care directly

**Comment by StreakyCobra** [comment]
Final point of this cleanup, closing this one :tada: :smile:

**Comment by d12frosted** [comment]
@StreakyCobra woo hoo :tada:
